client/locales/fr.json

```diff
@@ -148,6 +164,7 @@
             "map": "/map - vous enverra une simple image de la carte de votre maison.",
             "power": "/power - spécifie la puissance actuelle du ventilateur dans une plage de 1 à 100, ou une valeur spéciale 105 pour le mode vadrouille pour les robots qui le prennent en charge.\n\n38 - Silencieux\n60 - Équilibré\n75 - Turbo\n100 - Max (non recommandé!)",
             "powerV3": "/power - spécifie la puissance actuelle du ventilateur:\n\n101 - Calme\n102 - Equilibré\n103 - Turbo\n104 - Max\n105 - Lavette",
+            "segments": "/pièce - énumérera les noms de pièces disponibles qui pourraient être utilisés pour le nettoyage des pièces.",
```

Comment:
Currently all these commands are only in English without localization, so here it should be just "/rooms".
Probably they should be also localized one day...

Reply:
Ok, the correction was made